packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -2915,20 +4035,12 @@ test.serial('writeGranuleFromApi() given a granule with all fields populated is
     }
   );
 
-  // Validate that none of the responses come back as 'null', we want them removed, not set
```

Comment:
Why was this removed?    It's present in main/feature, and seems like a valid assertion given a quick glance.

Reply:
It is a duplicated assertion - see line 2944/4056. I was not sure which one to keep, opted for the latter one (let me know if that should change).

Reply:
Ah - got it, that totally makes sense.